community/portworx/values.yaml

```diff
@@ -1,5 +1,8 @@
 # Please uncomment and specify values for these options as per your requirements.
 
+generateSTCForMigration: false        # Only used for daemonset to operator migration.
```

Comment:
good point, i actually just changed it to true. which means by default helm will generate the stc. I am only aware of one parameter that is not handled by operator generated stc, which is the is-iks annotation set to true.

Reply:
hmm I mean if we should only rely on operator to do the operation not helm. In case of helm deployment, do we expect the customers not to change the daemonset spec manually?

Reply:
yes, customer should only give values.yaml, and we will generate stc. which is more reliable so far as i see.